### PR TITLE
Support relative commands in Paths

### PR DESCRIPTION
**Pull request by H1Gdev**
#### What does this implement/fix? Explain your changes.

- Save `Paths` as relative commands.
- Corresponds to dynamic changes in code.

### CODE REVIEW

File: Source/Paths/SvgArcSegment.cs [hunk 1]
```diff
@@ -43,24 +42,26 @@ private static double CalculateVectorAngle(double ux, double uy, double vx, doub
             return DoublePI - (ta - tb);
         }
 
-        public override void AddToPath(GraphicsPath graphicsPath)
+        public override PointF AddToPath(GraphicsPath graphicsPath, PointF start, SvgPathSegmentList parent)
```
Comment (mrbean-bremen) [inline Source/Paths/SvgArcSegment.cs:45]: same here

File: Source/Paths/SvgClosePathSegment.cs [hunk 2]
```diff
@@ -1,31 +1,32 @@
+﻿using System.Drawing;
 using System.Drawing.Drawing2D;
 
 namespace Svg.Pathing
 {
     public sealed class SvgClosePathSegment : SvgPathSegment
     {
-        public override void AddToPath(GraphicsPath graphicsPath)
+        public SvgClosePathSegment(bool isRelative)
```
Comment (mrbean-bremen) [inline Source/Paths/SvgClosePathSegment.cs:8]: Same here - make it default to true.

**Review by mrbean-bremen — COMMENTED** [review]
As far as I can see, this all looks good, and is a helpful addition, You also added a lot of tests. The only thing is that we don't want to change the public API without announcing it first - as I wrote, add the previous API back and mark it as obsolete, so we can remove it in a later version.

**Comment by H1Gdev** [comment]
I understand idea of wanting to keep previous API  for compatibility.
But that doesn't work perfectly correctly.(I didn't want to change public API if I could...)
Is that okay ??

File: Source/Paths/SvgClosePathSegment.cs [hunk 3]
```diff
@@ -1,31 +1,43 @@
...
 
         public override string ToString()
         {
-            return "z";
+            return IsRelative ? "z" : "Z";
+        }
+
+        [System.Obsolete("Use new constructor.")]
+        public SvgClosePathSegment()
+            : this(true)
```
Comment (H1Gdev) [inline Source/Paths/SvgClosePathSegment.cs:34]: We don't know why this command is only relative path...

**Review by mrbean-bremen — APPROVED** [review]
Thanks!

**Pull request merged**